libpq/README.md

```diff
@@ -0,0 +1,12 @@
+# Habitat Package: libpq
+The Habitat Maintainers humans@habitat.sh
+
+## Description
+Habitat package for the client side libraries of postgresql, including
+libpq and psql
```

Comment:
how about  "and the `psql` command-line client utility" to slightly aid the discoverability?